packages/create-cloudflare/templates/react-router/c3.ts

```diff
@@ -0,0 +1,51 @@
+import { logRaw } from "@cloudflare/cli";
+import { brandColor, dim } from "@cloudflare/cli/colors";
+import { runFrameworkGenerator } from "frameworks/index";
+import { detectPackageManager } from "helpers/packageManagers";
+import { installPackages } from "helpers/packages";
+import type { TemplateConfig } from "../../src/templates";
+import type { C3Context } from "types";
+
+const { npm } = detectPackageManager();
+
+const generate = async (ctx: C3Context) => {
+	await runFrameworkGenerator(ctx, [
+		ctx.project.name,
+		"--template",
+		"https://github.com/remix-run/react-router-templates/tree/main/cloudflare",
+	]);
+
+	logRaw(""); // newline
+};
+
+const configure = async () => {
+	await installPackages(["wrangler@latest"], {
+		dev: true,
+		startText: "Updating the Wrangler version",
+		doneText: `${brandColor(`updated`)} ${dim("wrangler@latest")}`,
+	});
+};
+
+const config: TemplateConfig = {
+	configVersion: 1,
+	id: "react-router",
+	platform: "workers",
+	frameworkCli: "create-react-router",
+	displayName: "React Router",
```

Comment:
```suggestion
	displayName: "React Router (formerly Remix)",
```

There was some discussion about making it clear that this is the successor to Remix and removing the Remix entry. I think we also decided to have `--framework=remix` direct to the React Router template if specified via the CLI.

@petebacondarwin Can you confirm?

Reply:
Yes I believe that was the decision. No interactive way to get Remix, but still supporting the `--framework` approach for backward compatibility of CI-type jobs.

Reply:
I'm a bit confused - should `--framework=remix` direct to remix or react router? if it directs react-router, does this mean the remix template is removed, since it can neither be accessed through cli or interactivity?

Reply:
Yep, that's my understanding. We remove the Remix template and make `remix` an alias to `react-router` when specified in the CLI.

Reply:
Sorry if I created confusion here. I don't believe that `--framework=remix` should alias to React Router. The main reason for keeping that option is for backward compatibility with users who are, for instance, running C3 as part of a non-interactive automated flow. In this case they should expect to get a Remix project; and may well be relying upon that. Removing the Remix template (equivalent to redirecting to React-Router) is arguably a breaking change.

So my understanding of what we should do is:

- hide the remix option from interactive diaglogs
- leave `--framework=remix` as-is

Reply:
Done here [49ba99b](https://github.com/cloudflare/workers-sdk/pull/8553/commits/49ba99bf7a79964aa7ca82587c745c46c883a1c4)